### PR TITLE
Fixed HW-54116: Transaction Details for Debit Transaction is showing the incorrect transaction amount

### PR DESCRIPTION
**Pull request by oleksii-lubianyi-epam**
Fixed tests

### CODE REVIEW

File: Tests/Receipt/ReceiptDetailPresenterTests.swift [hunk 1]
```diff
@@ -82,7 +82,7 @@ class ReceiptDetailPresenterTests: XCTestCase {
 
         XCTAssertTrue(rowEqual(section.rows[0], ("Amount:", "-9.87 USD")))
         XCTAssertTrue(rowEqual(section.rows[1], ("Fee:", "0.11 USD")))
-        XCTAssertTrue(rowEqual(section.rows[2], ("Transaction:", "-9.98 USD")))
+        XCTAssertTrue(rowEqual(section.rows[2], ("Transaction:", "-9.76 USD")))
```
Comment (jsaini-hw) [inline Tests/Receipt/ReceiptDetailPresenterTests.swift]: Transaction should not contain - or +

File: Tests/Receipt/ReceiptDetailPresenterTests.swift [hunk 2]
```diff
@@ -96,7 +96,7 @@ class ReceiptDetailPresenterTests: XCTestCase {
 
         XCTAssertTrue(rowEqual(section.rows[0], ("Amount:", "-100500 KRW")))
         XCTAssertTrue(rowEqual(section.rows[1], ("Fee:", "500 KRW")))
-        XCTAssertTrue(rowEqual(section.rows[2], ("Transaction:", "-101000 KRW")))
+        XCTAssertTrue(rowEqual(section.rows[2], ("Transaction:", "-100000 KRW")))
```
Comment (jsaini-hw) [inline Tests/Receipt/ReceiptDetailPresenterTests.swift]: same as above

File: UITests/Receipts/TransactionDetailTests.swift [hunk 3]
```diff
@@ -53,7 +53,7 @@ class TransactionDetailTests: BaseTests {
         verifyDetailSection(receiptIdVal: "55176991", dateVal: expectedDateValue, clientIdVal: nil)
 
         // FEE Section
-        verifyFeeSection(amountVal: "-5.00 USD", feeVal: "2.00 USD", transactionVal: "-7.00 USD")
+        verifyFeeSection(amountVal: "-5.00 USD", feeVal: "2.00 USD", transactionVal: "-3.00 USD")
```
Comment (jsaini-hw) [inline UITests/Receipts/TransactionDetailTests.swift]: same as above

**Comment by oleksii-lubianyi-epam** [comment]
Improper branch name. To be resubmitted with correct name

**Pull request closed without merging**